### PR TITLE
Add support for APDS-9960, Digital Proximity, Ambient Light, RGB and Gesture Sensor

### PR DESCRIPTION
**Pull request by alankrantas**
For #277.

This one is tricky since there are a lot of functionalities packed in this little thing. For now I simply use the Arduino version's default settings (pulse length, pulse count, gain, etc.) so that the gesture detection would work.

### CODE REVIEW

File: apds9960/apds9960_nano_33_ble.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+	"tinygo.org/x/drivers"
+)
+
+// New creates a new APDS-9960 connection. The I2C bus must already be
+// configured.
+//
+// This function only creates the Device object, it does not touch the device.
+func New(bus drivers.I2C) Device {
+	// turn on internal power pin (machine.P0_22) and I2C1 pullups power pin (machine.P1_00)
```
Comment (ysoldak) [inline apds9960/apds9960_nano_33_ble.go:18]: Comment does not match the code?

File: apds9960/apds9960_nano_33_ble.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	// and wait a moment.
+	ENV := machine.P0_15
```
Comment (ysoldak) [inline apds9960/apds9960_nano_33_ble.go]: I wonder, would it be good idea to extract these special pins to tinygo repo, in `board_nano-33-ble.go`?
Also, this initialisation sequence seem to be generic for this sensor, only difference from board to board is pins? If so, why not move it into `apds9960.Configure`? Much the same as wifinina driver does, for example.

**Review by ysoldak — COMMENTED** [review]
Thank you, for the PR!
I have Nano 33 BLE Sense board and a project in mind that involves proximity sensor. How timely!

File: apds9960/apds9960_nano_33_ble.go (hunk 1 above)
Comment (alankrantas) [reply apds9960/apds9960_nano_33_ble.go:18]: Which part? "it does not touch the device"? In a sense it is, because this only turns on the power supply.

Comment (ysoldak) [reply apds9960/apds9960_nano_33_ble.go:18]: No, I mean the pin numbers

Comment (alankrantas) [reply apds9960/apds9960_nano_33_ble.go:18]: Ok, I guess I mixed up the internal I2C pins.

**Comment by ysoldak** [comment]
Verified on my Nano 33 BLE Sense board, works!
Thank you for implementing this.

**Pull request merged**